src/subscription_manager/repofile.py

```diff
@@ -610,6 +610,10 @@ def fix_content(self, content):
         if zypper_cont['gpgkey'] in ['https://', 'http://']:
             del zypper_cont['gpgkey']
 
+        # make sure gpg key download doesn't fail because of private certs
+        if zypper_cont['gpgkey'] and sslverify and sslverify in ['1']:
+            zypper_cont['gpgkey'] += "?ssl_verify=host"
```

Comment:
I don't understand to the purpose of this change. The `gpgkey` in generated file looks like this:

```
gpgkey=file:/etc/pki/rpm-gpg/RPM-GPG-KEY-candlepin
```

Why should it be changed to this:

```
gpgkey=file:/etc/pki/rpm-gpg/RPM-GPG-KEY-candlepin?ssl_verify=host
```

Can you provide any link to documentation?

Reply:
in case of using katello, the URL is targeting http:// or https:// and then you want to tell zypper, to only validate the ssl based on the host as described in https://github.com/openSUSE/libzypp/blob/67f55b474d67f77c1868955da8542a7acfa70a9f/zypp/media/MediaManager.h#L394